dist/index.js

```diff
@@ -0,0 +1,2042 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
Presuming that we are going to publish this project as an NPM library, we wouldn't want to commit this file. Subsequent commits will conflict significantly with this file. And we'd want a build step to build this file before commit, so we might as well instead put that build step into `prepublish`.